Misanthropic-Mistletoes/reactive-nutshell
# Caroline one

# Description

*** Julian *** Check the add and delete function. The edit doesn't work yet. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. commit changes on your current branch
2. git checkout master
3. git fetch --all
4. git checkout caroline-one
5. code .
6. npm start
7. click events in nav bar
8. Check to be sure Delete works; I'm still working on Edit
9. The next event to complete should have a different background color and larger font



# Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
